packages/lexical-editor/src/ui/DropDown.tsx

```diff
@@ -162,42 +166,70 @@ export function DropDown({
 
     useEffect(() => {
         const button = buttonRef.current;
-        if (button === null || !showDropDown) {
-            return;
+        const dropDown = dropDownRef.current;
+
+        if (showDropDown && button !== null && dropDown !== null) {
+            dropDown.style.top = "44px";
+            dropDown.style.left = `${button.offsetLeft}px`;
+            setPositionIsCalculated(true);
         }
```

Comment:
```suggestion
     if (showDropDown && dropDown && button) {
            dropDown.style.top = "44px";
            dropDown.style.left = `${button.offsetLeft}px`;
            setPositionIsCalculated(true);
        }
```

A bit more elegant maybe? ⬆️